Sources/Apodini/Semantic Model Builder/gRPC/GRPCService/GRPCService.swift

```diff
@@ -23,6 +24,77 @@ class GRPCService {
         self.serviceName = name
         self.app = app.vapor.app
     }
+
+    internal func checkContentType(request: Vapor.Request) -> Bool {
+        let contentType = request.headers.first(name: .contentType)
+        switch contentType {
+        case "application/grpc", "application/grpc+proto":
```

Comment:
maybe we could rather create an enum for content types.

Reply:
https://github.com/vapor/vapor/blob/master/Sources/Vapor/HTTP/Headers/HTTPMediaType.swift

Reply:
Thanks for the link @tgymnich 🙌 I created a `grpc` media type constant and replaced the string comparisons with it.